README.md

```diff
@@ -141,19 +158,32 @@ $ pytest tests/test_file_name.py
 
 ... where `test_file_name.py` is relpaced with the correct test file name.
 
-## Project Write-Up: How to Complete and Submit
+To run a single test by name:
+
+```bash
+# Must be in activated virtual environment
+$ pytest tests/test_file_name.py::test_name
+```
 
-The goal of this project is to write code in `main.py` so that as many of the tests pass as possible.
+... where `test_name.py` is relpaced with the name of the function.
 
-To complete this project, use the above workflow and follow these steps:
+## Play Testing
 
-1. Start with making the tests in `test_wave_01.py` pass.
-1. Review your code in `main.py` and see if there are ways you can make the code more readable.
-1. Then, work on making the tests in `test_wave_02.py` pass.
-1. Review your code in `main.py`
-1. Repeat on all test files until submission time.
+While we will mainly use a Test Driven Development (TDD) workflow for this project, it can be helpful to run code independently from running tests. To do this, a file `play_tester.py` is provided.
 
-At submission time, no matter where you are, submit the project via Learn.
+To run this file, use:
+
+```bash
+python3 play_tester.py
+```
```

Comment:
This also has to be run in venv, might be worth adding `# Must be in activated virtual environment` NBD if it's overkill though